### PR TITLE
Vore the Naughty Words

### PR DESCRIPTION
**Pull request by jjpark-kb**
## About The Pull Request

Removes/replaces naughty words, so we don't violate Github's ToS.

## Why It's Good For The Game

Naughty words are bad

## Changelog
:cl:
code: removed/replaced naughty words
/:cl:

### CODE REVIEW

**Comment by PersianXerxes** [comment]
Removing "faggot" is a really shitty kneejerk reaction. :/

**Comment by hugmeorelse** [comment]
> Removing "faggot" is a really shitty kneejerk reaction. :/

I think it's ridiculous too, but considering what Github is doing, it's best to safe than sorry. :c

**Comment by PersianXerxes** [comment]
> 
> 
> > Removing "faggot" is a really shitty kneejerk reaction. :/
> 
> I think it's ridiculous too, but considering what Github is doing, it's best to safe than sorry. :c

It would be better to follow upstream's lead and wait until Oranges explains what Github said the ban was for, honestly.

**Comment by hugmeorelse** [comment]
> > > Removing "faggot" is a really shitty kneejerk reaction. :/
> > 
> > 
> > I think it's ridiculous too, but considering what Github is doing, it's best to safe than sorry. :c
> 
> It would be better to follow upstream's lead and wait until Oranges explains what Github said the ban was for, honestly.

That's why this won't be merged yet.

File: strings/names/death_commando.txt [hunk 1]
```diff
@@ -61,7 +61,7 @@ Splint Chesthair
 Stabby McGee
 Stump Beefgnaw
 Stump Chunkman
-THAT DAMN FAGGOT TRAITOR GEORGE MELONS
+THAT DAMN meatball TRAITOR GEORGE MELONS
```
Comment (Trilbyspaceclone) [inline strings/names/death_commando.txt:64]: This is more funny then before

**Comment by Commandersand** [comment]
why is Ramadan completely removed

**Comment by hugmeorelse** [comment]
> why is Ramadan completely removed

I mean, ignoring the current context, it is out of place for the server. We don't really have Chinese New Year or Hindu holidays either.

**Comment by CinderWC** [comment]
While i personally dont agree with any of these changes whatsoever it sadly is a measure we are possibly have to look into with Github's crackdown on TG lately.

On another note: I am surprised we have this many offensive stuffs in our code damn son

File: strings/ion_laws.json [hunk 2]
```diff
@@ -766,7 +766,7 @@
         "THIRTEEN SEQUELS",
         "THREE WISHES",
         "A SITCOM",
-        "THAT GRIEFING FAGGOT GEORGE MELONS",
+        "THAT GRIEFING meatball GEORGE MELONS",
```
Comment (CinderWC) [inline strings/ion_laws.json:769]: Atleast make the meatball part fullcaps as well to stick with the rest of the text, same for `death_commando.txt`

File: strings/names/death_commando.txt (hunk 1 above)
Comment (CinderWC) [inline strings/names/death_commando.txt:64]: Same here

File: strings/ion_laws.json (hunk 2 above)
Comment (jjpark-kb) [reply strings/ion_laws.json:769]: This is intentional. The sudden all lowercase on a singular word makes it obvious it once was something else. Perhaps if they look at the blame, they can find what it was, making it not only funny in game, but out of game as well.

File: strings/names/death_commando.txt (hunk 1 above)
Comment (jjpark-kb) [reply strings/names/death_commando.txt:64]: This is intentional. The sudden all lowercase on a singular word makes it obvious it once was something else. Perhaps if they look at the blame, they can find what it was, making it not only funny in game, but out of game as well.

Comment (jjpark-kb) [reply strings/names/death_commando.txt:64]: Thank you for appreciating my replacing humor.

File: strings/ion_laws.json (hunk 2 above)
Comment (hugmeorelse) [reply strings/ion_laws.json:769]: Hehe. 

Meatball.

**Comment by CoreFlare** [comment]
replace the name of the emoji too
icons/emoji.dmi

**Pull request closed without merging**